asserts/crypto.go

```diff
@@ -28,26 +28,77 @@ import (
 	"encoding/base64"
 	"encoding/hex"
 	"fmt"
+	"io"
 	"time"
 
 	"golang.org/x/crypto/openpgp/packet"
 )
 
-// TODO: eventually this should be the only non-test file using/importing directly from golang.org/x/crypto
+const (
+	maxEncodeLineLength = 76
+)
 
-func generatePrivateKey() (*packet.PrivateKey, error) {
-	priv, err := rsa.GenerateKey(rand.Reader, 2048)
+func encodeFormatAndData(format string, data []byte) []byte {
+	buf := bytes.NewBufferString(format)
+	buf.WriteByte(' ')
+	buf.Grow(base64.StdEncoding.EncodedLen(len(data)))
```

Comment:
Somewhat of a nitpick, but if we're growing, why are we doing that after doing operations that must have just re-allocated a buffer internally?  `Grow(enclen(data)+1+len(format))`?